example/dms-secret.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: dms-secret
+  namespace: dms-test
+type: Opaque
+data:
+  src_addr: c291cmNlLWRhdGFiYXNlLnRlc3QuY29t
+  src_port: MTQzMw==
+  src_ssl: cmVxdWlyZQ==
+  src_engine: c3Fsc2VydmVy
+  src_user: ZXhhbXBsZQ==
+  src_pass: ZXhhbXBsZQ==
+  src_database: QWR2ZW50dXJlV29ya3M=
+  dst_addr: ZGVzdGluYXRpb24tZGF0YWJhc2UudGVzdC5jb20=
+  dst_port: NTQzMg==
+  dst_ssl: cmVxdWlyZQ==
+  dst_engine: cG9zdGdyZXM=
+  dst_user: ZXhhbXBsZQ==
+  dst_pass: ZXhhbXBsZQ==
+  dst_database: QWR2ZW50dXJlV29ya3M=
```

Comment:
Is there anyway we can have Terraform create this secret? Instead of raw yaml that our pipeline will have to decrypt.